dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/resources/de/digitalcollections/cudami/server/backend/impl/database/migration/V2_7_0__DDL_Table_for_RenderingTemplates.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS rendering_templates (
+  id SERIAL PRIMARY KEY NOT NULL,
```

Comment:
ein id-feld hatten wir bisher nicht... und wird auch nirgends benutzt... ich weiß, dass das normalerweise in datenbanken angelegt wird und von der datenbank befüllt wird. sollen wir es drin lassen?

Reply:
Hab das ID-Feld entfernt und die UUID zum Primary Key gemacht.